doc/explanation/clustering.md

```diff
@@ -37,86 +48,101 @@ Automatic roles are assigned by LXD itself and cannot be modified by the user.
 | `event-hub`           | no            | Exchange point (hub) for the internal LXD events (requires at least two) |
 | `ovn-chassis`         | no            | Uplink gateway candidate for OVN networks |
 
-The default number of voting nodes is 3 and the default number of stand-by nodes is 2.
-This means that your cluster will remain operation as long as you switch off at most one voting node at a time.
+The default number of voter members ([`cluster.max_voters`](server)) is three.
+The default number of stand-by members ([`cluster.max_standby`](server)) is two.
+With this configuration, your cluster will remain operational as long as you switch off at most one voting member at a time.
 
-You can change the desired number of voting and stand-by nodes with:
+See {ref}`cluster-manage` for more information.
 
-```bash
-lxc config set cluster.max_voters <n>
-```
+(clustering-offline-members)=
+#### Offline members and fault tolerance
 
-and
+If a cluster member is down for more than the configured offline threshold, its status is marked as offline.
+In this case, no operations are possible on this member, and neither are operations that require a state change across all members.
 
-```bash
-lxc config set cluster.max_standby <n>
-```
+As soon as the offline member comes back online, operations are available again.
 
-with the constraint that the maximum number of voters must be odd and must be least 3, while the maximum number of stand-by nodes must be between 0 and 5.
+If the member that goes offline is the leader itself, the other members will elect a new leader.
 
-#### Offline nodes and fault tolerance
+If you can't or don't want to bring the server back online, you can [delete it from the cluster](cluster-manage-delete-members).
 
-If a node is down for more than 20 seconds, its status will be marked as OFFLINE and no operation will be possible on it, as well as operations that require a state change across all nodes.
+You can tweak the amount of seconds after which a non-responding member is considered offline by setting the [`cluster.offline_threshold`](server) configuration.
+The default value is 20 seconds.
+The minimum value is 10 seconds.
 
-If the node that goes offline is the leader itself, the other nodes will elect a new leader.
+See {ref}`cluster-recover` for more information.
 
-As soon as the offline node comes back online, operations will be available again.
+#### Failure domains
 
-If you can't or don't want to bring the node back online, you can delete it from the cluster using `lxc cluster remove --force <node name>`.
+You can use failure domains to indicate which cluster members should be given preference when assigning roles to a cluster member that has been shut down or has crashed.
```

Comment:
Rather than "that has been shut down or has crashed." here (which are some of the reasons a member may be considered offline) we could say "that has gone offline." as that would encompass all possibly causes of a member going offline (which is when a role change is initiated).